sos/plugins/__init__.py

```diff
@@ -963,11 +964,12 @@ def _make_command_filename(self, exe, subdir=None):
                              self._mangle_command(exe))
 
         # check for collisions
-        if os.path.exists(outfn):
+        if os.path.exists(os.path.join(self.archive.get_tmp_dir(), outfn)):
```

Comment:
Why does this even count from '2'? It's in the original, but why keep it broken? A more conventional approach would be like:

```
collision
collision.1
collision.2
```

Etc. Or in the event that we are running up against the limit:

```
alongcollision
alongcollisi.1
alongcollisi.2
...
```

Reply:
OK will update the PR accordingly; then the " - 5" won't be necessary.